File: Examples/Java/SDK/src/main/java/com/aspose/cells/cloud/examples/worksheet/GetColumn.java
package com.aspose.cells.cloud.examples.worksheet;

import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;

import java.io.IOException;
import java.nio.file.Path;

public class GetColumn {

	public static void main(String... args) throws IOException {
		try {
			// Instantiate Aspose Storage API SDK
			StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

			// Instantiate Aspose Words API SDK
			CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);
			String input = "Sample1.xlsx";
			Path inputFile = Utils.getPath(GetColumn.class, input);
			String sheet = "Sheet1";
			int index = 0;

			storageApi.PutCreate(input, null, Utils.STORAGE, inputFile.toFile());

			com.aspose.cells.model.ColumnResponse r = cellsApi
					.GetWorksheetColumn(inputFile.getFileName().toString(), sheet, index, Utils.STORAGE, null);

			System.out.println("Column: " + r.getColumn().getLink().getHref());
		}

		catch (Exception e) {
			e.printStackTrace();
		}
	}
}

File: Examples/DotNet/CSharp/Worksheet/GetColumn.cs
using Aspose.Cloud;
using System;
namespace Aspose.Cells.Cloud.Examples.Worksheet
{
    class GetColumn
    {
        static void Main()
        {
            string dataDir = Common.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);

            string input = "sample1.xlsx";
            string sheetName = "Sheet1";
            Common.StorageService.File.UploadFile(dataDir + input, input, storage: Common.STORAGE);

            WorksheetColumnResponse apiResponse = Common.CellsService.WorksheetColumns.ReadWorksheetColumnDataByColumnIndex(input, sheetName, 0, Common.FOLDER, storage: Common.STORAGE);

            Console.WriteLine(" Response: " + apiResponse.Column.IsHidden.ToString());
        }
    }
}

